src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -159,31 +159,6 @@ def test_input_equals_output(resource_client, input_model, output_model):
         pruned_output_model, pruned_input_model, resource_client.create_only_paths
     )
 
-    assertion_error_message = (
-        "All properties specified in the request MUST "
-        "be present in the model returned, and they MUST"
-        " match exactly, with the exception of properties"
-        " defined as writeOnlyProperties in the resource schema"
+    response_contains_resource_model_equal_current_model(
+        resource_client, pruned_input_model, pruned_output_model
```

Comment:
why dont you want to include whole method under decorate? so it could be used as annotation